exoctk/phase_constraint_overlap/phase_constraint_overlap.py

```diff
@@ -30,19 +37,37 @@
 import requests
 import urllib
 
+from astropy.time import Time
+
+from scipy import optimize
 from exoctk.utils import get_target_data
 
-def calculate_phase(period, obsDur, winSize):
-    ''' Function to calculate the min and max phase.
+def calculate_phase(period, preDur, winSize, t0 = None, ecc = None, omega = None, inc = None, secondary = False, winn_approx = False, get_tsec = False):
```

Comment:
I followed your suggestion for `preDur` (https://github.com/ExoCTK/exoctk/pull/357/commits/2d6d3be48b967cba3914adbbd18c7c8c719bcd57), `winSize` (https://github.com/ExoCTK/exoctk/pull/357/commits/026a963bd4ef5302fe4cf285f4de3feb16e4ecb5) and `get_tsec` (https://github.com/ExoCTK/exoctk/pull/357/commits/4b2155412ade0b467801f9e6d3a7dad8bf8c5d92). For the others I just kept them, because those are very common contractions for astronomers/physicists, and for non-astronomers/physicists my hunch is that `ecc` sounds as obscure as `eccentricity`, for example.